### PR TITLE
[Merged by Bors] - feat(Geometry/RingedSpace): Initial object in `LocallyRingedSpace`

### PR DESCRIPTION
**Pull request by erdOne**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/CategoryTheory/Limits/Shapes/StrictInitial.lean [hunk 1]
```diff
@@ -195,6 +202,13 @@ theorem IsTerminal.strict_hom_ext (hI : IsTerminal I) {A : C} (f g : I ⟶ A) :
   exact eq_of_inv_eq_inv (hI.hom_ext (inv f) (inv g))
 #align category_theory.limits.is_terminal.strict_hom_ext CategoryTheory.Limits.IsTerminal.strict_hom_ext
 
+/-- If `X ⟶ Y` with `Y` being a strict initial object, then `X` is also an initial object. -/
```
Comment (joelriou) [inline Mathlib/CategoryTheory/Limits/Shapes/StrictInitial.lean:205]:
```suggestion
/-- If `X ⟶ Y` with `Y` being a strict terminal object, then `X` is also an terminal object. -/
```

**Comment by joelriou** [comment]
Thanks!

bors d+

**Comment by erdOne** [comment]
maintainer merge

**Comment by erdOne** [comment]
bors merge

**Pull request closed without merging**